### ISSUE TITLE
The SSB transmission with TX ERS (remote) is produced on the air with a big delay (2 seconds ...see more) Sunsdr2 Pro Beta 1 and ERS server of 28.10.2022

### ISSUE THREAD

**Issue by f6ctt** [ExpertSDR3/ExpertSDR3-REMOTE-SYSTEM#8]

**Comment by ea5ccy** [comment]
We are going from bad to worse in the ERS. If in the last versions there was a Delay of 2 seconds in SSB now with the Beta 1.0.2 released today December 23rd when trying to start the remote software in windows 10 it closes itself again and appears busy in the device manager. 
I had to go back to the previous version which at least half works.

**Comment by sq6emm** [comment]
The problem still exists... Anyone is using SunSDR in remote way? This bug makes it unusable....

**Comment by ea5ccy** [comment]
Yes, the transmission delay issue is still ongoing with the latest Beta 1.0.3 release on 10 February 2023. The strange thing is that if you do FT8 using TCI there is no delay at all but on telephony we still have the 2 seconds delay. Hopefully the developers will fix it soon. We have been waiting for several months now.

**Comment by YO8TNB** [comment]
Unfortunately, the voice delay is still present in version 1.0.4 beta. I am frustrated that I can't use my voice in the SSB contests. This should have been a priority to be resolved, but unfortunately, this is not happening. I am despondent.

**Comment by themagoose** [comment]
Same thing on 10.4
Big delay in ssb (even if it SEEMS that the delay is A LITTLE less if compared to 10.3)
almost no delay on ft8
